src/ucp/wireup/wireup_cm.c

```diff
@@ -420,7 +420,15 @@ static void ucp_ep_cm_disconnect_flushed_cb(ucp_request_t *req)
     ucs_async_context_t *async = &ucp_ep->worker->async;
 
     UCS_ASYNC_BLOCK(async);
-    ucp_ep_cm_disconnect_cm_lane(ucp_ep);
+    ucs_assert(req->flags & UCP_REQUEST_FLAG_COMPLETED);
```

Comment:
how can we test this in gtest?

Reply:
need to add force close option on connection reset to gtests, do you prefer to add it in this PR?

Reply:
yes, let's have some gtest, it's small PR anyway

Reply:
ok, gtest change won't be so small :)